puzzlejs/puzzle.js

```diff
@@ -557,7 +557,7 @@ function PuzzleEntry(p, index) {
 
             if (this.options["data-text-shift-key"] == "none" && e.key.length == 1) { ch = e.key; }
 
-            if (this.options["data-text-characters"].includes(ch)) {
+            if (this.options["data-text-characters"].includes(ch.toUpperCase())) {
```

Comment:
Great catch, thanks! I'd prefer the toUpperCase() call to go on the line above, though, so there's zero change to the original code.